alacritty.yml

```diff
@@ -114,6 +110,10 @@ colors:
 
   # Colors the cursor will use if `custom_cursor_colors` is true
   cursor:
+    # Use custom cursor colors. If true, always display the cursor in the
+    # `colors.cursor.text` and `colors.cursor.cursor` colors,
+    # otherwise invert the colors of the background and foreground.
+    custom_colors: false
     text: '0x000000'
     cursor: '0xffffff'
```

Comment:
There's a part of me that finds it confusing to ship a config with values set that aren't used. I get that we are providing a complete set of config items to make it easy for users to modify, but maybe the `text` and `cursor` should be commented out?

Reply:
The idea of the config is to provide every option available with documentation. So the config is self-documenting without needing any other source.

If the user doesn't want something, it's always better to just remove it, rather than leaving it out in my opinion.